plugins/main/public/components/common/wazuh-discover/wz-flyout-discover.tsx

```diff
@@ -297,15 +299,15 @@ const WazuhFlyoutDiscoverComponent = (props: WazuhDiscoverProps) => {
                   showResetButton={false}
                   tooltip={
                     results?.hits?.total &&
-                    results?.hits?.total > MAX_ENTRIES_PER_QUERY
+                      results?.hits?.total > MAX_ENTRIES_PER_QUERY
                       ? {
-                          ariaLabel: 'Warning',
-                          content: `The query results has exceeded the limit of 10,000 hits. To provide a better experience the table only shows the first ${formatNumWithCommas(
-                            MAX_ENTRIES_PER_QUERY,
-                          )} hits.`,
-                          iconType: 'alert',
-                          position: 'top',
-                        }
+                        ariaLabel: 'Warning',
+                        content: `The query results has exceeded the limit of 10,000 hits. To provide a better experience the table only shows the first ${formatNumWithCommas(
```

Comment:
**nitpick**: the string has the limit number hardcoded on the string, if this value is stored in a constant, then we could use the constant instead. This ensures changing the constant affects to the string too.

Reply:
I saw you replaced another way of the string instead of the hard-coded `10,000` value. Could we use a constant instead?
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/34042064/f109ef9f-85c4-4eaa-9d1c-a1fcc4c9159b)